servers/SCsub

```diff
@@ -30,7 +30,9 @@ if not env["disable_3d"]:
     env.add_source_files(env.servers_sources, "physics_server_3d.cpp")
     env.add_source_files(env.servers_sources, "physics_server_3d_wrap_mt.cpp")
     SConscript("xr/SCsub")
```

Comment:
I would move this under the `if` below, and undo the changes in `servers/xr/SCsub`.

Reply:
I still have this suggestion pending, IMO it's cleaner to not process that SCsub if we're disabling the feature.

Aside from that I think it's good to go.